dump_manager_bmc.cpp

```diff
@@ -78,6 +115,14 @@ void Manager::createEntry(const uint32_t id, const std::string objPath,
     }
 }
 
+void Manager::createEntry(const uint32_t id, const std::string objPath,
+                          const uint64_t ms, uint64_t fileSize,
+                          const std::filesystem::path& file,
+                          phosphor::dump::OperationStatus status)
+{
+    createEntry(id, objPath, ms, fileSize, file, "", status);
+}
```

Comment:
This new function you can avoid with the use of below prototype

createEntry(const uint32_t id, const std::string objPath,
                          const uint64_t ms, uint64_t fileSize,
                          const std::filesystem::path& file,
                          phosphor::dump::OperationStatus status
                          std::string& generatorId = "" )